src/ApplicationTemplate.sln

```diff
@@ -11,6 +11,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "root", "root", "{8711E408-6
 		..\.azure-pipelines.yml = ..\.azure-pipelines.yml
 		..\.editorconfig = ..\.editorconfig
 		..\.gitignore = ..\.gitignore
+		..\APP_README.md = ..\APP_README.md
```

Comment:
Not sure you should add this because it gets renamed, right? Make sure you test with a generated project.

Reply:
You are right!